backend/src/GraphQl/Institutions/InstitutionMutations.cs

```diff
@@ -348,4 +348,44 @@ await context
         await context.SaveChangesAsync(cancellationToken).ConfigureAwait(false);
         return new DeleteInstitutionPayload();
     }
+
+   
+    public async Task<ChangeInstitutionOperatingStatePayload> ChangeInstitutionOperatingStateAsync(
+        ChangeInstitutionOperatingStateInput input,
+        ClaimsPrincipal claimsPrincipal,
+        [Service(ServiceKind.Resolver)] UserManager<User> userManager,
+        ApplicationDbContext context,
+        CancellationToken cancellationToken
+    )
+    {
+        // if (!await InstitutionAuthorization.IsAuthorizedToChangeInstitutionOperatingState(
```

Comment:
I suppose with the Login-System working again, see my comment https://github.com/building-envelope-data/metabase/commit/52d82726b3df764e579447e0ddb38c6c9d50b421#commitcomment-143039216, you can uncomment this.